### ISSUE TITLE
Dynamic Voxel Resolution

### ISSUE THREAD

**Issue by DigitalLibrarian** [DigitalLibrarian/VS2010Projects#6]
The engine needs to support :
1. Many different "things" (terrain, player, vehicles, plants, clouds) not having the same resolution of voxels
2. Re-resolution-ing of entire chunks at run time.

The Chunk class is meant to be the basic unit of voxels.  That's where re-resolution-ing should go.  Maintaining set of different resolution actors will be different use cases for the Chunk class.

Acceptance Criteria:
- A chunk of voxels can be handled with any single float scale.

To "re-resolution" we'd need an arbitrary precision model, there is another ticket for that - #33

**Comment by DigitalLibrarian** [comment]
First off, I don't think everything will work if voxels aren't unit sized. _Most_ things will, but some heavy testing is needed here.

**Comment by DigitalLibrarian** [comment]
This depends on #17

**Comment by DigitalLibrarian** [comment]
#17 makes this redundant.  Closing.
